.github/workflows/stats.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request Stats
+
+on:
+  pull_request:
+    types: [opened]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.PR_REVIEW_PAT }}
+
+jobs:
+  stats:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
I think these permissions only affect the token in `GITHUB_TOKEN` (the default token). So they won't have any effect since we use `token: ${{ secrets.PR_REVIEW_PAT }}`.
And reading [the docs](https://github.com/flowwer-dev/pull-request-stats) it sounds to me like we only need the specific personal access token if we want to have stats across the whole organization?
Maybe try with the default token first (basically remove lines 8 and 19) and try again?

Reply:
AFAICT, it might be Gituhb like caching the workflow because I was spamming or something? When I add just these lines, the set of permissions still doesn't show up properly. I also realize now for the most part I can just re-run the workflow, but unsure how I can make it clear out